vpd-tool/include/tool_utils.hpp

```diff
@@ -254,5 +255,104 @@ inline void displayOnConsole(const std::string& i_fruPath,
 
     printJson(l_resultInJson);
 }
+
+/**
+ * @brief API to write keyword's value.
+ *
+ * This API writes keyword's value by requesting DBus service(vpd-manager) who
+ * hosts the 'WriteKeyword' method to update keyword's value.
+ *
+ * @param[in] i_vpdPath - EEPROM or object path, where keyword is present.
+ * @param[in] i_paramsToWriteData - Data required to update keyword's value.
+ *
+ * @return - Number of bytes written on success, -1 on failure.
+ *
+ * @throw - std::runtime_error, sdbusplus::exception::SdBusError
+ */
+inline int writeKeyword(const std::string& i_vpdPath,
+                        const types::WriteVpdParams i_paramsToWriteData)
+{
+    if (i_vpdPath.empty())
+    {
+        throw std::runtime_error("Empty path");
+    }
+
+    try
+    {
+        int l_rc = constants::FAILURE;
+        auto l_bus = sdbusplus::bus::new_default();
+
+        auto l_method = l_bus.new_method_call(
+            constants::vpdManagerService, constants::vpdManagerObjectPath,
+            constants::vpdManagerInfName, "WriteKeyword");
+
+        l_method.append(i_vpdPath, i_paramsToWriteData);
+        auto l_result = l_bus.call(l_method);
+
+        l_result.read(l_rc);
+
+        if (l_rc > 0)
+        {
+            std::cout << "Data updated successfully " << std::endl;
```

Comment:
Does this trace need to go under "verbose" option?

Reply:
This log needs to be printed anyways. This is confirmation log.